.travis.yml

```diff
@@ -37,6 +37,7 @@ matrix:
         - ./node_modules/.bin/eslint ./js/scripts_similarity.js
         - ./node_modules/.bin/eslint ./js/scripts_dendrogram.js
         - ./node_modules/.bin/eslint ./js/scripts_rwanalysis.js
+        - ./node_modules/.bin/eslint ./js/scripts_tokenizer.js
```

Comment:
Why not use a regex to match all the scripts?

Reply:
I think we haven't made all the files to pass EsLint yet...

Reply:
Ohohoh, yeah, sure.

Reply:
Nope, no need to sorry, things need to be done one at a time. I understand.